lib/common/widgets/disabled_main_button.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:fv_chat/common/styles/app_button_styles.dart';
+
+class DisabledMainButton extends StatelessWidget {
```

Comment:
no need to do disabled button like separate widget. it's the one of the main_button state.

if you set onPressed: null, your button is disabled. google this